exporter/importables.go

```diff
@@ -534,10 +535,38 @@ var resourcesMap map[string]importable = map[string]importable{
 		},
 	},
 	"databricks_cluster_policy": {
-		Service: "compute",
+		Service: "policies",
 		Name: func(ic *importContext, d *schema.ResourceData) string {
 			return d.Get("name").(string)
 		},
+		List: func(ic *importContext) error {
+			w, err := ic.Client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			policies, err := w.ClusterPolicies.ListAll(ic.Context, compute.ListClusterPoliciesRequest{})
```

Comment:
Why was listing all policies not required before?

Reply:
It was an initial design decision from the beginning of development, but now customers are starting to ask for it. The new implementation can behave the old way - just don't provide `policies` in the `-listing` switch